packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -133,7 +144,8 @@ contract L1Block is ISemver, IGasToken {
     ///   7. _blobBaseFee        L1 blob base fee.
     ///   8. _hash               L1 blockhash.
     ///   9. _batcherHash        Versioned hash to authenticate batcher by.
-    function setL1BlockValuesEcotone() external {
+    ///   10. _isDeposit         isDeposit flag
+    function setL1BlockValuesInterop() external {
```

Comment:
we can name this `setL1BlockValuesIsthmus` - it'll be more forward compatible because that is the hardfork that interop will activate - https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/overview.md

Reply:
https://github.com/ethereum-optimism/optimism/blob/0d78e5a6a433e30d350cdaadbc018d4b803fefaf/packages/contracts-bedrock/src/libraries/Encoding.sol#L188

there is also this library which I'm not sure what it does.